.github/workflows/linter.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Lint Code Base
         uses: github/super-linter@v3
         env:
-          VALIDATE_ALL_CODEBASE: false
+          VALIDATE_ALL_CODEBASE: true
```

Comment:
both VALIDATE_ALL_CODEBASE and VALIDATE_JAVASCRIPT_ES is default to true in the document, should we add here?
https://github.com/github/super-linter